README.md

```diff
@@ -1,194 +1,64 @@
-# Apify SDK monorepo
+# Actor SDK
```

Comment:
this will be monorepo for actor sdk + more (apify-cli, apify-client, storage-local, ...)

Reply:
Oof, right... then we cannot share a readme for all of them (or we *can* but it'd be noisy for each module). Do you want me to instead make a readme in the apify module?

Reply:
We can deal with that when we add more packages, right now we can use the root readme.

Reply:
Well if I add monorepo to the readme that won't make much sense on npm's side... Your call, I can add it back

Reply:
Yeah I guess we should have separete readme per package here, still the root one can talk about actor sdk in more detail than just "this is a monorepo, actor sdk is one of the packages", especially now :]

Reply:
Then should I copy this current readme into the apify module and write up an actor sdk monorepo readme?

Reply:
Yop. Btw the root one is beging copied via `copy.ts` script.